tests/test_ewmh.py

```diff
@@ -124,6 +124,38 @@ def test_wm_state_type(hlwm, x11):
     assert len(prop.value) == 2
 
 
+def test_ewmh_focus_client(hlwm, x11):
+    hlwm.call('set focus_stealing_prevention off')
+    # add another client that has the focus on the other tag
+    _, winid_focus = x11.create_client()
+    handle, winid = x11.create_client()
+    assert hlwm.get_attr('clients.focus.winid') == winid_focus
+
+    x11.ewmh.setActiveWindow(handle)
+    x11.display.flush()
+
+    assert hlwm.get_attr('clients.focus.winid') == winid
+
+
+@pytest.mark.parametrize('on_another_monitor', [True, False])
+def test_ewmh_focus_client_on_other_tag(hlwm, x11, on_another_monitor):
+    hlwm.call('set focus_stealing_prevention off')
+    hlwm.call('add tag2')
+    if on_another_monitor:  # if the tag shall be on another monitor
+        hlwm.call('add_monitor 800x600+600+0')
+    hlwm.call('rule tag=tag2 focus=off')
+    # add another client that has the focus on the other tag
+    x11.create_client()
+    handle, winid = x11.create_client()
+    assert 'focus' not in hlwm.list_children('clients')
+
+    x11.ewmh.setActiveWindow(handle)
```

Comment:
The name "handle" makes it really hard to understand, which client is meant (same in the test case above).
I cannot think of a better name, but client_one and client_two would be an improvement.

Reply:
This comment just made me notice that we have an `x11.create_client()` that returns the window ID as a second value and a `hlwm.create_client()` that returns it as the first window ID. Maybe we should make that consistent.

Reply:
> This comment just made me notice that we have an `x11.create_client()` that returns the window ID as a second value and a `hlwm.create_client()` that returns it as the first window ID. Maybe we should make that consistent.

This confuses me as well a lot.